CONTRIBUTING.rst

```diff
@@ -3,24 +3,20 @@ Contributing guide
 
 Thanks for your interest in contributing to cpplint.
 
-Any kinds of contributions are welcome: Bug reports, Documentation, Patches.
+Any kinds of contributions are welcome: Bug reports, Documentation, Patches. If you need some ideas, you may check out some of the tasks below:
 
-However, cpplint is a bit special as a project because it aims to closely follow what Google does in the upstream repository.
-That means Google remains the source of all major requirements and functionality of cpplint whereas this fork adds extensions to cpplint run on more environments and in more companies.
-The difference between this cpplint and Google should remain so small that anyone can at a glance see there is no added change that could be regarded as a security vulnerability.
+* Reducing interference with other popular tools
+* Update cpplint for modern python versions
+  * Converting setup.py to a pyproject.toml
+* Porting cpplint to other environments
+  * Making tests, especially clitest, work on Windows
+* Complete tickets in our `issue tracker <https://github.com/cpplint/cpplint/issues>`_
 
-Please consider our goals and non-goals below before contributing.
-
-Goals:
-
-* Provides cpplint as a PyPI package for multiple python versions
```

Comment:
why remove this goal?

IMO the goals vs non-goals section here was fine.

Reply:
We already provide it as a PyPI package, and “multiple python versions” is covered by other suggested contributions